src/context/AppContext.js

```diff
@@ -123,6 +125,26 @@ function getRoutes(user) {
         !user ||
         user.policy.organization !== undefined,
     },
+    {
+      name: 'Stakeholders',
+      linkTo: '/stakeholders',
+      component: Stakeholder,
+      icon: AccountTreeIcon,
+      //TODO this is temporary, need to add stakeholders policy
+      // disabled: false,
+      disabled:
+        process.env.REACT_APP_ENABLE_STAKEHOLDERS !== 'true' ||
+        !hasPermission(user, [
+          POLICIES.SUPER_PERMISSION,
+          POLICIES.APPROVE_TREE,
+          // POLICIES.LIST_STAKEHOLDERS,
```

Comment:
New policies have been assigned to the _Stakeholder Manager_ role in the dev, test and prod databases and can be assigned to users.